php/settings/settings-fields.php

```diff
@@ -215,6 +215,20 @@ function get_settings_fields() {
 			'default'    => true,
 			'codemirror' => 'styleActiveLine',
 		],
+		'keymap' => [
+			'name'       => __( 'Keymap', 'code-snippets' ),
+			'type'       => 'select',
+			'desc'       => __( 'The keymap to use in the editor.', 'code-snippets' ),
+			'default'    => 'default',
+			'options'    => [
+				'default' => __( 'Default', 'code-snippets' ),
+				'vim'     => __( 'Vim', 'code-snippets' ),
```

Comment:
Do you think these 3 are worth adding to the translation? Perhaps just having 'Default' is fine.